### DISCUSSION TITLE
Update Vision Statement

### DISCUSSION THREAD

**Discussion by AlecWhite** [cataclysmbn/Cataclysm-BN#3684] [General]
### Is your feature request related to a problem? Please describe.

Now that BN has got some traction I think it would help to update the game's [Vision Statement](https://gamescrye.com/blog/why-your-game-needs-a-vision-statement/) and maybe work in a [Game Design Document](https://en.wikipedia.org/wiki/Game_design_document) and this is taking into account the monolithic task of fixing the [game engine](https://github.com/cataclysmbnteam/Cataclysm-BN/issues/3143) that is still pending.

I think it would help a lot to clarify how "roguelike"/"roguelite and "sandbox" the game is meant to be. And to differentiate it more from DDA besides the vague "it has less realism and has more sci-fi stuff".

### Describe the solution you'd like

Make an updated Vision Statement for the future of Bright Nights.

### Additional context

I make this after the Nth time we had a discussion on Discord about what is the intended path that BN is taking.

**Comment by scarf005** [comment]
> [!Warning]
> This is my own opinion and **does not** represent the direction of the devteam.

to follow the [vision statement](https://gamescrye.com/blog/why-your-game-needs-a-vision-statement/):

> ## UI,UX,DX oriented lightweight fork of CDDA

## Player Experience (UI/UX)

- better support for mouse control
- QOL features over realism
- customizable gameplay

## Developer Experience (DX)

### For modders

- procedural item crafting system to cut down JSON bloat
- Lua scripting system to make modding powerful and easier
- mod manager

### For developers

- strip down game engine and delegate its hardcoded behavior to Lua hooks
- make builds easier and faster
- cut down total game code
- improve documentation
